src/Propel/Generator/Builder/Om/ObjectBuilder.php

```diff
@@ -282,6 +282,14 @@ protected function addClassBody(&$script)
             '\Propel\Runtime\Map\TableMap'
         );
 
+        $baseClass = $this->getBaseClass();
+        if (strrpos($baseClass, '\\') !== FALSE) {
+            $classesToDeclare[] = $baseClass;
+        }
+
+        $reflectionMethod = new \ReflectionMethod($this, 'declareClasses');
```

Comment:
Well, I'd completely not use any of those stuff.

```php
$baseClass = $this->getBaseClass();
if (false !=== strrpos($baseClass, '\\')) {
    $this->declareClasses($baseClass);
}
```

Doing it this way is easier to read, less modification and no user_func-call